Lean4Axiomatic/Natural/Exponentiation.lean

```diff
@@ -32,26 +33,28 @@ instance (priority := default+1) pow_inst
   pow := Exponentiation.Ops._pow
 }
 
-/-- Properties of exponentiation to a natural number. -/
+/-- Properties of exponentiation for a monoid type α to a natural number. -/
 class Exponentiation.Props
-    {α : Type} {ℕ : outParam Type} (mul : outParam (α → α → α))
-    [Core ℕ] [Addition ℕ] [Multiplication ℕ] [EqvOp α] [OfNat α 1] [Ops α ℕ]
+    {α : Type} {ℕ : outParam Type}
+    [Core ℕ] [Addition ℕ] [Multiplication ℕ] [EqvOp α] [Ops α ℕ]
+    [isMonoid : CA.Monoid.Monoid α]
```

Comment:
In almost all cases, instance arguments don't need to be named. I think that's true here too, can you remove the `isMonoid` name?